tests/cupyx_tests/scipy_tests/sparse_tests/test_linalg.py

```diff
@@ -1268,3 +1268,31 @@ def test_spilu_0(self, dtype, xp, sp):
         else:
             ainv = sp.linalg.spilu(a)
         return ainv.solve(b)
+
+
+@testing.parameterize(*testing.product({
+    'damp': [0.0, 1.0, 2.0],
+    'format': ['coo', 'csr', 'csc'],
+    'm': [30, 40, 50],
+    'n': [20, 30, 40, 50]
+}))
+class TestLsmr(unittest.TestCase):
+
+    def _make_matrix(self, sp):
+        A = testing.shaped_sparse_random((self.m, self.n), sp, density=0.2)
+        return A
+
+    def _make_normalized_vector(self, xp):
+        b = testing.shaped_random((self.m,), xp, scale=1)
+        return b
+
+    def _test_lsmr(self, xp, sp, a, b):
+        return sp.linalg.lsmr(a, b, damp=self.damp, atol=1e-6, btol=1e-6)
```

Comment:
It seems most of `lsmr` call finish with `istop=7` because of the small matrix size, where `maxiter=min([m,n])`. Would you explicitly give it a larger `maxiter`?

Reply:
I think we should make `maxiter = min([m,n]) * 10` similar to what we have for `cg` and `gmres`.